Tests.Integration.Presentation.Web/Tools/OrganizationGridConfigurationTestHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net;
+using System.Threading.Tasks;
+using Core.DomainModel.Organization;
+using Core.DomainModel;
+using Presentation.Web.Models.API.V1;
+using Presentation.Web.Models.API.V2.Internal.Response.Organizations;
+
+namespace Tests.Integration.Presentation.Web.Tools
+{
+    internal class OrganizationGridConfigurationTestHelper
+    {
+
+        private static string CreatePath(Guid orgUuid, string operation, OverviewType overviewType = OverviewType.ItSystemUsage)
+        {
+            return $"api/v2/internal/organizations/{orgUuid:D}/grid-configuration/{overviewType}/{operation}";
+        }
+
+        public static async Task<HttpResponseMessage> SendGetConfigurationRequestAsync(Guid orgUuid,
+            Cookie cookie = null)
+        {
+
+            var url = TestEnvironment.CreateUrl(CreatePath(orgUuid, "get"));
+            var httpCookie = cookie ?? await HttpApi.GetCookieAsync(OrganizationRole.LocalAdmin);
+            return await HttpApi.GetWithCookieAsync(url, httpCookie);
+        }
+
+        public static async Task<HttpResponseMessage> SendSaveConfigurationRequestAsync(Guid orgUuid, IEnumerable<KendoColumnConfigurationDTO> columns,
```

Comment:
Create a SaveConfigurationRequestAsync method that reuses this one, checks if the response has OK code, and reads the response body

Reply:
(it feels more "clean" to do that)

Reply:
Like this, or? 
![image](https://github.com/user-attachments/assets/119b15f6-0d27-4fae-85a9-e9c49450d1d5)

Reply:
and then change the return type to OrganizationGridConfigurationResponseDTO and return a response.ReadResponseBodyAsAsync<OrganizationGridConfigurationResponseDTO>();

so you can use the response for a check